mewarpx/mewarpx/plotting.py

```diff
@@ -0,0 +1,633 @@
+import matplotlib.pyplot as plt
+import matplotlib.colors as colors
+import numpy as np
+import copy
+import collections
+from mewarpx.mwxrun import mwxrun
+
+
+class ArrayPlot(object):
+
+    """Initialize and plot the passed array data. Many processing steps."""
+
+    param_defaults = {
+        "points": None,
+        "numpoints": 7,
+        "sweepaxlabel": None,
+        "xlabel": None,
+        "ylabel": None,
+        "title": None,
+        "titlestr": "Array plot",
+        "titleunits": "",
+        "titleline2": None,
+        'scale': 'symlog',
+        # Set to None to use adaptive linthresh
+        "linthresh": 0.2,
+        # If using adaptive linthresh, the minimum value it can take
+        "_linthreshmin": 1,
+        # If using adaptive linthresh, the percentile it should take
+        "_linthreshper": 5,
+        "linportion": 0.33,
+        "ncontours": 100,
+        "multiplier": 1.0,
+        "zeroinitial": False,
+        "plottype": 'phi',
+        "repeat": False,
+        "xaxis": 'z',
+        "yaxis": 'x',
+        "slicepos": None,
+        "labelsize": 24,
+        "legendsize": 12,
+        "titlesize": 32,
+        "ncontour_lines": 15,
+        "cmap": 'inferno',
+        "default_ticks": False,
+        "draw_cbar": True,
+        "draw_surface": False,
+        "draw_image": False,
+        "draw_contourlines": True,
```

Comment:
I think we both agree these are better defaults, so let's go ahead and change them now.
```suggestion
        "draw_image": True,
        "draw_contourlines": False,
```